### PR TITLE
Updating with latest whatsnew

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) fixes links and adds PRs from internal repo.

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [ ] Magento Open Source only
- [ ] Magento Commerce only

Is this update specific to an installed feature extension

- [ ] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/whats-new.html

### CODE REVIEW

File: src/_data/whats-new.yml [hunk 1]
```diff
@@ -107,20 +107,56 @@ entries:
...
   type: New topic
   date: July 28, 2020
   link: https://github.com/magento/merchdocs/pull/507
-- description: Inventory Management now supports [bundle products](https://docs.magento.com/user_guide/catalog/inventory-about-product-types.html).
+- description: Inventory Management now supports [bundle products](https://docs.magento.com/user-guide/catalog/inventory-about-product-types.html).
   versions: 2.4.0
   type: Major update
   date: July 28, 2020
   link: https://github.com/magento/merchdocs/pull/451
+
```
Comment (leslietilling) [inline src/_data/whats-new.yml:127]: Does this extra line matter?

File: src/_data/whats-new.yml [hunk 2]
```diff
@@ -107,20 +107,56 @@ entries:
...
+  versions: 2.4.0
+  type: Major update
+  date: July 28, 2020
+  link: 
+- description: Page Builder [Rows](https://docs.magento.com/m2/ee/user_guide/cms/page-builder-layout-row.html), [Banners](https://docs.magento.com/m2/ee/user_guide/cms/page-builder-media-banner.html), and [Sliders](https://docs.magento.com/m2/ee/user_guide/cms/page-builder-media-slider.html) now have the option to set their heights to the full-height of the page using a number with any CSS unit (px, %, vh, em) or a calculation between units (100vh - 237px).
+  versions: 2.4.0
+  type: Major update
+  date: July 28, 2020
+  link: 
+- description: Page Builder now has [templates](https://docs.magento.com/m2/ee/user_guide/cms/page-builder-templates.html) that can be created from existing content and applied to new content areas.
```
Comment (leslietilling) [inline src/_data/whats-new.yml:143]: This uses an old URL. 

```suggestion
- description: Page Builder now has [templates](https://docs.magento.com/user-guide/cms/page-builder-templates.html) that can be created from existing content and applied to new content areas.
```

File: src/_data/whats-new.yml [hunk 3]
```diff
@@ -107,20 +107,56 @@ entries:
...
+  versions: 2.4.0
+  type: Major update
+  date: July 28, 2020
+  link: 
+- description: Page Builder [Rows](https://docs.magento.com/m2/ee/user_guide/cms/page-builder-layout-row.html), [Banners](https://docs.magento.com/m2/ee/user_guide/cms/page-builder-media-banner.html), and [Sliders](https://docs.magento.com/m2/ee/user_guide/cms/page-builder-media-slider.html) now have the option to use videos for their backgrounds.
+  versions: 2.4.0
+  type: Major update
+  date: July 28, 2020
+  link: 
+- description: Page Builder [Rows](https://docs.magento.com/m2/ee/user_guide/cms/page-builder-layout-row.html), [Banners](https://docs.magento.com/m2/ee/user_guide/cms/page-builder-media-banner.html), and [Sliders](https://docs.magento.com/m2/ee/user_guide/cms/page-builder-media-slider.html) now have the option to set their heights to the full-height of the page using a number with any CSS unit (px, %, vh, em) or a calculation between units (100vh - 237px).
```
Comment (leslietilling) [inline src/_data/whats-new.yml:138]: This uses an old URL

```suggestion
- description: Page Builder [Rows](https://docs.magento.com/user-guide/cms/page-builder-layout-row.html), [Banners](https://docs.magento.com/user-guide/cms/page-builder-media-banner.html), and [Sliders](https://docs.magento.com/user-guide/cms/page-builder-media-slider.html) now have the option to set their heights to the full-height of the page using a number with any CSS unit (px, %, vh, em) or a calculation between units (100vh - 237px).
```

File: src/_data/whats-new.yml [hunk 4]
```diff
@@ -107,20 +107,56 @@ entries:
...
   type: Major update
   date: July 28, 2020
   link: https://github.com/magento/merchdocs/pull/451
+
+- description: All instances of Magento 2.4.0 must use Elasticsearch 7.6 as the catalog search engine. In previous releases, MySQL was the default catalog search engine, but it is no longer supported in this capacity. [Configuring Catalog Search](https://docs.magento.com/user-guide/catalog/search-configuration.html) describes the configuration fields for Elasticsearch 7.6.
+  versions: 2.4.0
+  type: Major update
+  date: July 28, 2020
+  link: 
+- description: Page Builder [Rows](https://docs.magento.com/m2/ee/user_guide/cms/page-builder-layout-row.html), [Banners](https://docs.magento.com/m2/ee/user_guide/cms/page-builder-media-banner.html), and [Sliders](https://docs.magento.com/m2/ee/user_guide/cms/page-builder-media-slider.html) now have the option to use videos for their backgrounds.
```
Comment (leslietilling) [inline src/_data/whats-new.yml:133]: This uses old URL

```suggestion
- description: Page Builder [Rows](https://docs.magento.com/user-guide/cms/page-builder-layout-row.html), [Banners](https://docs.magento.com/user-guide/cms/page-builder-media-banner.html), and [Sliders](https://docs.magento.com/user-guide/cms/page-builder-media-slider.html) now have the option to use videos for their backgrounds.
```

**Comment by erikmarr** [comment]
running tests

**Pull request merged**